InventoryBuilder.py

```diff
@@ -127,6 +127,8 @@ def create_resource_objects(self, vrops, token):
                         print("Collecting Cluster: " + cl_object.name)
                     cl_object.add_host()
                     for hs_object in cl_object.hosts:
+                        if hs_object.name != "node004-bb052.cc.ap-cn-1.cloud.sap":
+                            continue 
```

Comment:
That's why the test fails. There is no host resource with this name in the test. So it does not create datastores or virtual machines and fails.